Lib/test/test_typing.py

```diff
@@ -3251,14 +3251,14 @@ def x(self): pass
         class EmptyProtocol(Protocol): pass
 
         @runtime_checkable
-        class SupportsStartsWith(Protocol):
+        class SupportsStartswith(Protocol):
```

Comment:
Not sure why this name was changed, could you undo changes to this file?

Reply:
Good catch. I initially wrote all this for `assertStartsWith`. Then I replaced "With" with "with". Then I replaced back "with" with "With", but used more strict patterns to avoid false positives, so some names left changed.